dfastmi/gui/dialog_view_model.py

```diff
@@ -350,9 +352,14 @@ def _initialize_qthreshold(self):
         """
         Initialize the threshold discharge.
         """
-        if self.model.qthreshold < self.current_reach.qstagnant:
-            self.model.qthreshold = self.current_reach.qstagnant
-        self.qthreshold = self.model.qthreshold
+        self._qthreshold = 0.0
+        if (self.current_branch, self.current_reach) in self._qthreshold_cache:
+            self.qthreshold = max(
+                self._qthreshold_cache[(self.current_branch, self.current_reach)],
+                self.current_reach.qstagnant,
+            )
+        else:
+            self.qthreshold = self.current_reach.qstagnant
```

Comment:
so the model shouldn't be updated anymore?

Reply:
It does that in the property

Reply:
Understood resolving converstation.